modules/telegraf/main.tf

```diff
@@ -0,0 +1,9 @@
+resource "consul_key_prefix" "core_integration" {
```

Comment:
Might want to document how this module could be used with example templates.

Reply:
Okay sorry was missing the `README.md`, added in.